build.gradle

```diff
@@ -986,7 +986,14 @@ if (jdkVersion == '8') {
 }
 
 project(':iceberg-spark3') {
+  apply plugin: 'scala'
+
   sourceSets {
+    main {
+      // Use Scala Compiler for everything so we can do cross compilation within the module
```

Comment:
I don't think I understand why we can't use the default scala plugin settings. This is disabling Java compilation by removing srcDirs. Should we just remove the Java plugin?